mentoring/templates/html/mentoring_attempts.html

```diff
@@ -0,0 +1,5 @@
+<script type="text/template" id="xblock-attempts-template">
+  <% if (_.isNumber(max_attempts) && max_attempts > 0) {{ %>
+  <span> You have used <%= _.min([num_attempts, max_attempts]) %> of <%= max_attempts %> submissions.</span>
```

Comment:
Is it possible for this to read 'You have used 0 of 1 submissions'? Or 'You have used 1 of 1 submissions?'